doc/examples/setup_assistant/setup_assistant_tutorial.rst

```diff
@@ -115,35 +106,50 @@ robot in a plane.
 
 .. image:: setup_assistant_panda_virtual_joints.png
    :width: 700px
+   :align: center
+
+The virtual joint feature is especially useful when we have a robot that is attached to a mobile base,
+such as a mobile manipulator. In such cases, the virtual joint allows us to model the motion
+of the robot's base, which is important for motion planning and control.
+
 
 Step 4: Add Planning Groups
 ---------------------------
 
-Planning groups are used for semantically describing different parts
-of your robot, such as defining what an arm is, or an end effector.
+Planning groups in MoveIt semantically describe different parts of the robot, 
+such as the arm or end effector, to facilitate motion planning.
+
+A move group can be configured to correspond to a specific kinematic chain on the robot,
+which is a set of links and joints that define a sequence of transformations from the base of the robot to the end effector.
+For example, a move group might be defined to represent the arm of a robot, which would consist of all the links and joints necessary to move the arm.
+
+Move groups can also be represented by sets of links or joints corresponding on the robot.
+For example, a move group might be defined to represent the gripper of a robot, which would consist of all the links or joints necessary to move together to realize a gripper opening motion.
 
 * Click on the *Planning Groups* pane selector.
 
 * Click on *Add Group* and you should see the following screen:
 
-.. image:: setup_assistant_panda_planning_groups.png
+.. image:: planning_groups/setup_assistant_panda_planning_groups.png
    :width: 700px
+   :align: center
 
-Add the arm
+Add the arm group
 
 * We will first add Panda arm as a planning group
 
   * Enter *Group Name* as **panda_arm**
 
   * Choose *kdl_kinematics_plugin/KDLKinematicsPlugin* as the
     kinematics solver. *Note: if you have a custom robot and would
-    like a powerful custom IK solver, see* :doc:`Kinematics/IKFast </doc/examples/ikfast/ikfast_tutorial>`
+    like a powerful custom IK solver, see* `Kinematics/IKFast <https://moveit.picknik.ai/humble/doc/examples/ikfast/ikfast_tutorial.html>`_
```

Comment:
Please continue using `:doc:` tags as these will automatically pick the right version. Do not hard-code a humble specific link.